tests/test.rfb.js

```diff
@@ -517,6 +517,143 @@ describe('Remote Frame Buffer Protocol Client', function () {
         });
     });
 
+    describe('Caps Lock and Num Lock remote fixup', function () {
+        let client;
+        beforeEach(function () {
+            client = makeRFB();
+            sinon.stub(client, 'sendKey');
+        });
+
+        it('should toggle caps lock if remote caps lock is on and local is off', function () {
+            client._remoteCapsLock = true;
```

Comment:
Ideally the same thing here. Get the tests in to the correct state by faking the right messages.